### PR TITLE
Fix unit error in CCCVcharging

### PR DESCRIPTION
**Pull request by henrikt-ma**

### CODE REVIEW

**Review by hubertus65 — APPROVED** [review]

**Review by AHaumer — DISMISSED** [review]
I'd suggest the following change:
`  parameter Modelica.Units.SI.Current Isc = 1200 "Short-circuit current of cell at OCVmax";`
Why don't make this parameter public?

File: Modelica/Electrical/Batteries/Examples/CCCVcharging.mo
Comment (HansOlsson) [inline Modelica/Electrical/Batteries/Examples/CCCVcharging.mo:11]:
```suggestion
    Ri=cellData.OCVmax/Isc,
```
Together with other change from @AHaumer

File: Modelica/Electrical/Batteries/Examples/CCCVcharging.mo
Comment (henrikt-ma) [reply Modelica/Electrical/Batteries/Examples/CCCVcharging.mo:6]: Yes, why not.

**Comment by HansOlsson** [comment]
Coordinate as described in #4101

File: Modelica/Electrical/Batteries/Examples/CCCVcharging.mo
Comment (christiankral) [inline Modelica/Electrical/Batteries/Examples/CCCVcharging.mo:6]:
```suggestion
```

**Review by christiankral — COMMENTED** [review]
@henrikt-ma I suggest to remove `public`, see suggestion. 

Other than that, I am in favor of approving this proposal.

**Review by christiankral — APPROVED** [review]
`OCVmax` and `Isc` are now considered in each battery example

File: Modelica/Electrical/Batteries/Examples/CCCV_Cell.mo
Comment (maltelenz) [inline Modelica/Electrical/Batteries/Examples/CCCV_Cell.mo:5]: Match indentation to the remaining lines?
```suggestion
  parameter Modelica.Units.SI.Current Isc = 1200 "Short-circuit current of cell at OCVmax";
```

Comment (christiankral) [reply Modelica/Electrical/Batteries/Examples/CCCV_Cell.mo:5]: Remove blanks

File: Modelica/Electrical/Batteries/Examples/CCCV_StackRC.mo
Comment (HansOlsson) [inline Modelica/Electrical/Batteries/Examples/CCCV_StackRC.mo:15]:
```suggestion
    Ri=2*cellDataDegraded.OCVmax/Isc,
```
Previously it was 2*4.2/1200, and OCVmax=4.2, Isc=1200; but the factor 2 was lost.
Without this change the I don't see any difference between original and degraded.

**Review by HansOlsson — COMMENTED** [review]
Fine by me - I think it is better to wait for actual expert approval; and I haven't figured out how to remove myself as reviewer.

File: Modelica/Electrical/Batteries/Examples/CCCVcharging.mo
Comment (hubertus65) [reply Modelica/Electrical/Batteries/Examples/CCCVcharging.mo]: Resolved since outdated and agreed in meeting 2023-11-14

**Review by hubertus65 — APPROVED** [review]

**Pull request merged**
